src/views/AddItem.jsx

```diff
@@ -25,7 +36,7 @@ export function AddItem({ listToken }) {
 
 	const handleName = (e) => {
 		setSuccess(false);
-
+		setDuplicateError(false);
```

Comment:
I really like how you reset on handle change and we don't have to worry about timeouts ✨